fetch.js

```diff
@@ -311,7 +311,13 @@
       xhr.onload = function() {
         var status = (xhr.status === 1223) ? 204 : xhr.status
         if (status < 100 || status > 599) {
```

Comment:
False. This happens with React Native. The specific instance is if the
method is POST and the content type is not specified. The status is "0" and
that field is filled in with the error message. Without the description
included here, the client has no idea what the error is.
On Sep 29, 2015 2:42 PM, "David Graham" <notifications@github.com> wrote:

> In fetch.js
> <https://github.com/github/fetch/pull/208#discussion_r40734068>:
>
> > @@ -311,7 +311,13 @@
> >        xhr.onload = function() {
> >          var status = (xhr.status === 1223) ? 204 : xhr.status
> >          if (status < 100 || status > 599) {
>
> This check is in place only for IE 9, which can set xhr.status to invalid
> values. Unless another browser is reaching this block unintentionally, I
> think we can close this pull request.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/github/fetch/pull/208/files#r40734068>.
>